src/Surfnet/ServiceProviderDashboard/Application/ViewObject/EntityActions.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/**
+ * Copyright 2018 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Application\ViewObject;
+
+class EntityActions
```

Comment:
Nice separation of concerns!
Maybe it's an idea to only have one EntityAction per env/state and initialize it in the constructor of the entity to keep things more clear.